server/index.js

```diff
@@ -58,12 +61,63 @@ app.post(routes.API.TELEMMETRY, function (req, res) {
  * @param  {} routes.API.USER_SEARCH This is the user search api url
  * @param  {} res
  * @param  {} {res.status(200) Sending the api response
- * @param  {[]}} .json({users}) Sending empty array response 
+ * @param  {[]}} .json({users}) Sending empty array response
  */
 app.post(routes.API.USER_SEARCH, function (req, res) {
   res.status(200).json({ users: [] })
 });
 
+/**
+ * @param  {} routes.API.USER_ROLE This is the user role api url
+ * @param  {} res
+ * @param  {} {res.status(200) Sending the api response
+ * @param  {[]}} .json({userRoles}) Sending array response
+ */
+ app.get(routes.API.USER_ROLE, function (req, res) {
+  let response = responseUtils.successResponse({
+    apiId: "api.v1.roles",
+    apiVersion: "1.0",
+    msgid: uuid(),
+    result: envVariables.USER_ROLE});
```

Comment:
Still, this didn't work if the env variable is set, have you tested this API by setting up env?